File: Answers/05_if/e03_leapYear.cs
using System;

class Program
{
    static void Main()
    {
        int year = int.Parse(Console.ReadLine());

        if ((year % 4 == 0 && year % 100 != 0) || (year % 400 == 0))
        {
            Console.WriteLine($"Rok {year} je prestupný.");
        }
        else
        {
            Console.WriteLine($"Rok {year} nie je prestupný.");
        }
    }
}

File: Answers/05_if/e03_leapYear.py
rok = int(input("Zadajte rok: "))

if (rok % 4 == 0 and rok % 100 != 0) or (rok % 400 == 0):
    print(f"Rok {rok} je prestupný.")
else:
    print(f"Rok {rok} nie je prestupný.")